src/ExadelMentorship.BusinessLogic/Features/WeatherFeature/FutureWeather/FutureWeatherCommandResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using ExadelMentorship.BusinessLogic.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ExadelMentorship.BusinessLogic.Features.WeatherFeature.FutureWeather
+{
+    public class FutureWeatherCommandResponse
```

Comment:
Now, as you can see, having this class is no use anymore. You can get rid of it and as a response generic type, use directly `IEnumerable<City>`.

Reply:
done